content/pages/constitution.md

```diff
@@ -168,9 +170,33 @@ computing.
     2. To design all media circulated by the committee including: posters,
        social media profiles and the official website.
 
-All committee memebers are free to delegate their responsibilities to others as
-as they see fit, but are ultimately responsible for the completion of their
-assigned duties.
+9.  All committee memebers are free to delegate their responsibilities to others
+    as they see fit, but are ultimately responsible for the completion of their
+    assigned duties.
+
+10. All committee members are responsible to support their fellow members and
+    ensure their assigned duties are being completed. If a member notices that
+    a fellow member has fell behind on their responsibilities, the supporting
+    member shall:
+
+    1. Reach out to the member who has fallen behind in private, and inquire
```

Comment:
edit to: _"Reach out to the member who has fallen behind in private, and inquire about the responsibility in question. If they are not comfortable with this, they can bring the issue to the Chair or Secretary, who will take over for steps 2-4"_